### PR TITLE
add i18n support

### PR DESCRIPTION
**Pull request by kkzzhizhou**
add i18n support

### CODE REVIEW

**Comment by SuperChamp234** [comment]
Hey @kkzzhizhou ! We appreciate your effort into helping with habitica-sync, can you explain your PR in brief?

**Comment by kkzzhizhou** [comment]
this pr is for the suport all other language use the react-i18next.
it can change language as the obsidian set.

**Comment by SuperChamp234** [comment]
Hey @kkzzhizhou !

You should open an another pull request for adding/removing tasks! And can you please re-base your code to the `dev` branch instead of main?

File: src/view/App.tsx [hunk 1]
```diff
@@ -51,15 +52,11 @@ class App extends React.Component<any, any> {
         let cronDate = new Date(lastCron);
         let now = new Date();
         if (cronDate.getDate() != now.getDate() || (cronDate.getMonth() != now.getMonth() || cronDate.getFullYear() != now.getFullYear())) {
-            // return (
```
Comment (SuperChamp234) [inline src/view/App.tsx:54]: Can you please not remove the existing cron implementation? We fixed it in commit d10a2ca747c1ada5e255bef6d39eddc45b0dcf26 Atleast provide a reason for it.

**Comment by SuperChamp234** [comment]
Hey @kkzzhizhou !
I've run your code and it runs great so far! I must say I have a few problems with it
image
The buttons as you see, take up a huge chunk of space in the existing UI and makes it a lot clunky. The window is only meant as a quick widget to view tasks and I don't think allowing edits/new tasks should be done there. We were thinking of adding a modal that is triggered from the command palette that allows us to add/edit tasks. So, I would like you to move your add, delete and edit tasks commits to a new pull request so that we can work on them!

**Comment by SuperChamp234** [comment]
Also, can you rebase onto our dev branch? That's the branch we push the commits to before releasing.

**Comment by kkzzhizhou** [comment]
ok, i will close this pr the request only add a i18n pr

File: src/view/App.tsx (hunk 1 above)
Comment (kkzzhizhou) [reply src/view/App.tsx:54]: ok，got it.

**Pull request closed without merging**